README.md

```diff
@@ -25,6 +25,49 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
 
 For game compatibility, see [community compatibility feedback](https://report.ppsspp.org/games).
 
+What's new in 1.16
+==================
+- RetroAchievements support ([#17589], [#17631], many more). See [RetroAchievements on ppsspp.org](https://www.ppsspp.org/docs/reference/retro-achievements).
+- New JIT backends:
+  - RISC-V, based on IR, by unknownbrackets
+  - x86 based on IR, by unknownbrackets. May end up faster than the existing one.
+- Input fixes
+  - Improve behavior when analog and digital inputs clash ([#17960])
+  - Combo mapping is now disabled by default ([#17673])
+  - Android: Better tracking of devices names ([#17562], auto config)
+  - Fix mapping custom touch buttons to analog inputs ([#17433])
+- Rendering performance and fixes
+  - Fix flicker in WWE Smackdown vs Raw 2006 ([#18009]), video flicker in Naruto 2 ([#18008])
+  - Fix bad colors in Syphon Filter: Logan's Shadow menu ([#17965])
+  - On lower-end, revert shaders to be less "uber" ([#17449], [#17922])
```

Comment:
Might qualify with something like `On lower-end devices, avoid "uber" shaders due to driver bugs` or something.

-[Unknown]